install.sh

```diff
@@ -194,94 +195,95 @@ __install_installer_deps() {
 	done
 }
 
-__install_bro() {
+__install_zeek() {
 
-
-	if  [ "$_OS" == "Ubuntu" ] && [ "$_OS_CODENAME" == "xenial" -o "$_OS_CODENAME" == "trusty" ] ; then
-		# Bro 2.6 will not compile under Debian 8/ Ubuntu Trusty/ Xenial in the OpenSuse Build Service
-		# Install Bro 2.5.5
-		__add_deb_repo "deb http://download.opensuse.org/repositories/home:/logan_bhis:/branches:/network:/bro/xUbuntu_$(lsb_release -rs)/ /" \
-			"Bro" \
-			"https://download.opensuse.org/repositories/home:logan_bhis:branches:network:bro/xUbuntu_$(lsb_release -rs)/Release.key"
+	if  [ "$_OS" == "Ubuntu" ] && [ "$_OS_CODENAME" == "xenial" ] ; then
+		# The Zeek Project does not host a package repo for Ubuntu 16.04.
+		# Use Security Onion's version of Zeek.
+		sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 9373E47F9BF8216D23D32BBBE1E6759023F386C7
+		__add_deb_repo "deb http://ppa.launchpad.net/securityonion/stable/ubuntu xenial main" \
+		"securityonion-ubuntu-stable-xenial"
+		__install_packages securityonion-bro
+		ln -s /opt/bro /opt/zeek
 	else
 		case "$_OS" in
 			Ubuntu)
-				__add_deb_repo "deb http://download.opensuse.org/repositories/home:/logan_bhis:/branches:/network:/bro-2-6/xUbuntu_$(lsb_release -rs)/ /" \
-					"Bro" \
-					"https://download.opensuse.org/repositories/home:logan_bhis:branches:network:bro-2-6/xUbuntu_$(lsb_release -rs)/Release.key"
+				__add_deb_repo "deb [ arch=$(dpkg --print-architecture) ] http://download.opensuse.org/repositories/security:/zeek/xUbuntu_$(lsb_release -rs)/ /" \
+				"security:zeek" \
+				"https://download.opensuse.org/repositories/security:/zeek/xUbuntu_$(lsb_release -rs)/Release.key"
 				;;
 			CentOS|RedHatEnterprise|RedHatEnterpriseServer)
-				__add_rpm_repo https://download.opensuse.org/repositories/home:logan_bhis:branches:network:bro-2-6/CentOS_7/home:logan_bhis:branches:network:bro-2-6.repo
+				__add_rpm_repo "https://download.opensuse.org/repositories/security:zeek/CentOS_7/security:zeek.repo"
```

Comment:
I think this is missing a slash:
`https://download.opensuse.org/repositories/security:zeek/` -> `https://download.opensuse.org/repositories/security:/zeek/`

When I go to it in my browser I get redirected anyway so it might not matter. But the Ubuntu URL above has a slash too.

Reply:
Good catch, I saw this fail in a VM. Must have forgotten to transfer the change!

Reply:
Done.